src/main/webapp/app/view/project/admin/users/project-user-invite-delete-dialog.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="row justify-content-center">
+    <div class="col-10">
+        <form *ngIf="invitation" name="deleteForm" (ngSubmit)="confirmDelete(invitation?.id!)">
+            <div class="modal-header">
+                <h2 class="modal-title" jhiTranslate="entity.delete.title">Confirm delete operation</h2>
+            </div>
+
+            <div class="modal-body">
+                <jhi-alert-error></jhi-alert-error>
+
+                <p id="jhi-delete-invitation-heading" jhiTranslate="eventPlannerApp.invitation.delete.question" [translateValues]="{ name: invitation.email }">Are you sure you want to delete this Invitation?</p>
```

Comment:
`invitation` is optional, no need for any annotations?

Reply:
was meinst du mit annotations?

Reply:
ah du meinst ein ngIf guard? der ist weiter oben